### ISSUE TITLE
Error handler does not support `InvalidInput`, `InvalidActor`, or `InvalidState` errors

### ISSUE THREAD

**Issue by timolegros** [hicommonwealth/commonwealth#9405]
## Description
In the commands/query framework we throw 3 types of errors:
- `InvalidInput`: equivalent to AppError
- `InvalidState`: equivalent to ServerError
- `InvalidActor`: authentication error (no explicit equivalent just errors thrown by passport)

These errors are not properly supported by `setupErrorHandlers.ts`. For example, if you set up a command that throws an `InvalidInput` error and integrate it using the express adapter, the error will be returned as an HTTP 500 error. We need to map the above errors to relevant error codes in `setupErrorHandlers.ts`

**Comment by Rotorsoft** [comment]
We have error middleware in `/libs/adapters/express/middleware.ts`, which can be added to the integration routers using the express adapter.... or do we need to merge this with the legacy error handlers?

**Comment by timolegros** [comment]
> We have error middleware in `/libs/adapters/express/middleware.ts`, which can be added to the integration routers using the express adapter.... or do we need to merge this with the legacy error handlers?

I hadn't seen that middleware. The middleware that currently applies is `setupErrorHandlers` because it handles errors globally across all routes on the express app. Is there a reason this error handling middleware is not applied to any of the new express routers?

**Comment by Rotorsoft** [comment]
yep... it was never wired to any router. I can take this ticket and basically merge it to the legacy `setupErrorHandlers`.
